spreadsheet/workbook_test.go

```diff
@@ -408,3 +408,19 @@ func TestCopySheetByName(t *testing.T) {
 		t.Fatalf("expected sheets count %d, got %d", wasCount+1, wb.SheetCount())
 	}
 }
+
+func TestExtractText(t *testing.T) {
+	testFilePath := "./testdata/extraction.xlsx"
+
+	want := "qweasdzxcsdfgdfgcvbdgddfgcbn123556434"
+
+	got, err := spreadsheet.ExtractText(testFilePath)
```

Comment:
What happens if there are many worksheets?  Is it possible to extract from specific worksheets?
I guess we can add to the ExtractText documentation an explanation of how it works and comment that it is pretty rudimentary, getting entire text of whole spreadsheet.
Would be good to explain what it does and how to use it, if there are more complex needs people can raise issues and we can design more powerful extraction in the future.

Reply:
I added an arg to represent sheets indexes. Probably would be a good idea to add an arg representing sheets names, to be able to choose from 2 versions of extraction. But it will break the interface which may be created some day, since we well have 2 funcs with different signature here. It needs to be discussed